models/migrations/migrations.go

```diff
@@ -278,11 +279,98 @@ func dropTableColumns(sess *xorm.Session, tableName string, columnNames ...strin
 	if tableName == "" || len(columnNames) == 0 {
 		return nil
 	}
+	// TODO: This will not work if there are foreign keys
```

Comment:
Doesn't xorm create no foreign keys? (= that case would never happen)

Reply:
Currently xorm does not create foreign keys

Reply:
I just put the warning in so that if we ever do try to create foreign keys we are alerted that this will break